JASP-Tests/R/tests/testthat/test-networkanalysis.R

```diff
@@ -0,0 +1,43 @@
+context("Network Analysis")
+
+# does not test
+# - error handling
+
+test_that("All tables results match", {
+  options <- JASPTools::analysisOptions("NetworkAnalysis")
+  options$variables <- c("contNormal", "contcor1", "contcor2")
+  options$tableCentrality <- TRUE
+  options$tableClustering <- TRUE
+  options$tableWeightsMatrix <- TRUE
+  options$tableLayout <- TRUE
+  results <- JASPTools::run("NetworkAnalysis", "debug.csv", options, view=FALSE, quiet=TRUE, , sideEffects = "pkgLoading")
+
+  table <- results[["results"]][["generalTB"]][["data"]]
+  expect_equal_tables(table,
+    list("Network", 3, "2 / 3", 0.333333333333333)
+  )
+
+  table <- results[["results"]][["centralityTB"]][["data"]]
+  expect_equal_tables(table,
+    list("contNormal", -0.577350269189626, -1.12003079401266, -1.14291478283658,
+      "contcor1", 1.15470053837925, 0.803219560925303, 0.713968266021615,
+      "contcor2", -0.577350269189626, 0.316811233087358, 0.428946516814968)
+  )
+
+  table <- results[["results"]][["clusteringTB"]][["data"]]
+  expect_equal_tables(table,
+    list("contcor1", 0, 0, 0, 0, "contcor2", 0, 0, 0, 0, "contNormal",
```

Comment:
I get a different result for this: `list("contNormal", 0, 0, 0, 0, "contcor1", 0, 0, 0, 0, "contcor2",
 0, 0, 0, 0)`